python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -35,6 +35,48 @@ def load_temp_for_hash_compare(
     )
 
 
+def update_prev_next_trip_stop(db_manager: DatabaseManager) -> None:
+    """
+    Update `previous_trip_stop_pk_id` and `next_trip_stop_pk_id` fields in `vehicle_events` table
+    """
+    prev_next_trip_stop_cte = (
+        sa.select(
+            VehicleEvents.pk_id,
+            sa.func.lead(VehicleEvents.pk_id)
+            .over(
+                partition_by=VehicleEvents.trip_hash,
+                order_by=VehicleEvents.stop_sequence,
+            )
+            .label("next_trip_stop_pk_id"),
+            sa.func.lag(VehicleEvents.pk_id)
+            .over(
+                partition_by=VehicleEvents.trip_hash,
+                order_by=VehicleEvents.stop_sequence,
+            )
+            .label("previous_trip_stop_pk_id"),
+        ).join(
+            TempHashCompare,
+            TempHashCompare.hash == VehicleEvents.trip_hash,
+        )
+    ).cte(name="prev_next_trip_stops")
```

Comment:
Update `previous_trip_stop_pk_id` and `next_trip_stop_pk_id` for any records associated with new `trip_hash` values loaded into rds.